color_windows.go

```diff
@@ -0,0 +1,17 @@
+package color
+
+import (
+	"os"
+
+	"golang.org/x/sys/windows"
+)
+
+func init() {
+	// Opt-in for ansi color support for current process.
+	// https://learn.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences#output-sequences
+	var outMode uint32
+	out := windows.Handle(os.Stdout.Fd())
+	if err := windows.GetConsoleMode(out, &outMode); err == nil {
+		_ = windows.SetConsoleMode(out, outMode|windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING)
```

Comment:
Shouldn't `ENABLE_PROCESSED_OUTPUT` be set as well?

> Characters written by the [WriteFile](https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-writefile) or [WriteConsole](https://learn.microsoft.com/en-us/windows/console/writeconsole) function or echoed by the [ReadFile](https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-readfile) or [ReadConsole](https://learn.microsoft.com/en-us/windows/console/readconsole) function are parsed for ASCII control sequences, and the correct action is performed. Backspace, tab, bell, carriage return, and line feed characters are processed. It should be enabled when using control sequences or when ENABLE_VIRTUAL_TERMINAL_PROCESSING is set.

Reference: https://learn.microsoft.com/en-us/windows/console/setconsolemode

Reply:
> Any thought if ENABLE_PROCESSED_OUTPUT should be set as well?

I don't know. It was not needed in order to get color output in git bash.

Reply:
It looks we need to set `ENABLE_PROCESSED_OUTPUT`, from the docs:

![ScreenShot-2023-03-01-at-10 05 20@2x](https://user-images.githubusercontent.com/438920/222067982-e374db7e-8365-4c79-b75c-3e753fe8b2cd.jpg)

Reply:
@martinlindhe could you add this flag as well? Let's make sure to follow the doc rules, there are probably some edge cases we're not aware yet, but it's probably only fixed if `ENABLE_PROCESSED_OUTPUT` is set.

Reply:
@fatih Fixed in #186